library/core/class.oauth2.php

```diff
@@ -256,7 +278,17 @@ public function getProviderKey() {
         return $this->providerKey;
     }
 
-
+    /**
+     * Register a call back function so that multiple plugins can use it as an entry point on SSO
```

Comment:
Missing period at end of sentence. Should also have a blank comment line after this / before the long description.